entries/r/rippling.com.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Rippling": {
+    "domain": "rippling.com",
+    "url": "https://www.rippling.com/",
```

Comment:
The `url` tag isn't needed if the domain redirects to the same page as the value of the `url` tag.
```suggestion
```